one_fm/one_fm/custom/job_applicant.json

```diff
@@ -4991,37 +5051,37 @@
    "collapsible": 0,
    "collapsible_depends_on": null,
    "columns": 0,
-   "creation": "2020-12-08 20:25:57.159078",
+   "creation": "2020-11-30 21:46:48.303658",
    "default": null,
-   "depends_on": "eval:doc.one_fm_applicant_is_overseas_or_local == 'Overseas'",
+   "depends_on": null,
    "description": null,
    "docstatus": 0,
    "dt": "Job Applicant",
-   "fetch_from": null,
+   "fetch_from": "job_title.designation",
    "fetch_if_empty": 0,
-   "fieldname": "one_fm_country_of_overseas",
+   "fieldname": "one_fm_designation",
```

Comment:
We removed this field, since in ERPNext core of Job Applicant have a designation field